api/src/main/java/org/infinispan/api/sync/SyncCache.java

```diff
@@ -42,7 +45,40 @@
     * @param key
     * @return the value
     */
-   V get(K key);
+   default V get(K key) {
```

Comment:
Missing (comparing with Async)
* `findContinuously`
* `remove(K key, CacheEntryVersion version)`

Reply:
I've moved `findContinuously` to `SyncQuery`.
Added the `remove` method